openmc/data/data.py

```diff
@@ -332,6 +333,34 @@ def decay_constant(isotope):
     return _LOG_TWO / t if t else 0.0
 
 
+def decay_energy(isotope):
+    """Return decay energy of isotope in eV or 0.0 if isotope is stable
+
+    Decay energy values are from the `ENDF/B-VIII.0 decay sublibrary
+    <https://www.nndc.bnl.gov/endf-b8.0/download.html>`_.
+
+    .. versionadded:: 0.14.1
+
+    Parameters
+    ----------
+    isotope : str
+        Name of isotope, e.g., 'U235'
+
+    Returns
+    -------
+    float
+        Decay energy of isotope in [eV]
+
+    """
+    global _DECAY_ENERGY
+    if not _DECAY_ENERGY:
+        # Load ENDF/B-VIII.0 data from JSON file
+        decay_energy_path = Path(__file__).with_name('decay_energy.json')
+        _DECAY_ENERGY = json.loads(decay_energy_path.read_text())
+    t = half_life(isotope)
+    return _DECAY_ENERGY.get(isotope.lower()) if t else 0.0
```

Comment:
I would agree with @shimwell's suggestion here. I think we should rely on the chain file for getting these values rather than having another JSON file that we're storing in the repository. You can take a look at the `decay_photon_energy` function to see how it is set up using the chain file (found from `openmc.config`).

Reply:
Thanks for your comments, the chain file is used to get decay energy just like `decay_photon_energy` function.